charts/kube-prometheus-stack/values.yaml

```diff
@@ -5300,8 +5300,10 @@ thanosRuler:
 ##
 cleanPrometheusOperatorObjectNames: false
 
-## Extra manifests to deploy as an array
-extraManifests: []
+## Extra manifests to deploy.  Can be of type dict or list.
+## If dict, keys are ignored and only values are used.
+## Items contained within extraObjects can be defined as dict or string and are passed through tpl.
+extraManifests: null
```

Comment:
Also please add a list entry below, which is an string and contains helm expressions, e.g. an if.

Reply:
I think I covered what you asked for.  Happy to make further changes if you spot something.